<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMenu</class>
 <widget class="QWidget" name="GameMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Игровое Меню</string>
  </property>
  <widget class="QPushButton" name="Exit_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Play_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Играть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reset_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обнулить</string>
   </property>
  </widget>
  <widget class="QLabel" name="Main_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Farmer's Valley</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
